src/app/core/templates/users/user-create/user-create.component.html

```diff
@@ -74,7 +74,8 @@ <h1>Hey {{firstName ? firstName : "you"}}!<br>
                 <div class="col-xs-12 col-md-8 col-lg-6">
 
                     <div class="form-group">
-                        <label for="userCreate-major" class="col-xs-12 col-sm-3 control-label">TU/e Major</label>
+                        <label for="userCreate-major" class="col-xs-12 col-sm-3 control-label">
+                            <span *ngIf="membership == memberships[1] || membership == memberships[2]">(old) </span>TU/e Major</label>
```

Comment:
Might be nicer to check against the actual value, as changing the array layout wouldn't result in unexpected behaviour